package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "flowcards-playground",
+  "version": "1.0.0",
```

Should I use `alpha`, `beta` and `0.x.x` until we agree is a stable version?

---

it depends of how are you going to bump the versions, do you have any idea? most probably it's going to be always 1.0.1-bla-bla, so i don't thinks it's necessary to create an alpha. Not big deal.

---

Why `1.0.1-bla-bla`, there is no build process in this repository right?

---

Do not set a version. Semantic Release Github Action will manage it.

```suggestion
  "version": "0.0.0-development",
```